tests/ImageTest.php

```diff
@@ -51,7 +51,7 @@ public function returns_correct_image_data()
         $this->assertEquals('49275 B', $image->sizeInBytesPostfix(' '));
         $this->assertEquals(48.12, $data['size_in_kilobytes']);
         $this->assertEquals('48.12KB', $data['size_in_kilobytes_postfix']);
-        $this->assertEquals('48.12 KB', $image->sizeInKiloBytesPostfix(3, ' '));
+        $this->assertEquals('48.120 KB', $image->sizeInKiloBytesPostfix(3, ' '));
```

Comment:
Once the `number_format` returns the `string` type, the expected value will be `48.120`.

Reply:
Did it work ok with it returning actual `float`?

Reply:
Ok. Let these methods return type hint is `:float`.

And the expected value will be `48.12 KB` again.

Reply:
Ok - let's do that.

Reply:
@sebastiansulinski, it will be done via [fa9178c](https://github.com/sebastiansulinski/file/pull/2/commits/fa9178cddc79bed74bb89120b07d08b77c6373f5).

Reply:
But just the `Size` class please.